pkg/pipelines/tekton/pipelines_integration_test.go

```diff
@@ -57,18 +55,22 @@ func TestOnClusterBuild(t *testing.T) {
 
 			pp := tekton.NewPipelinesProvider(
 				tekton.WithCredentialsProvider(credentialsProvider),
-				tekton.WithNamespace(ns),
-				tekton.WithProgressListener(pl{urlChan: urlChan}))
+				tekton.WithNamespace(ns))
 
 			f := createSimpleGoProject(t, ns)
 			f.Build.Builder = test.Builder
 
 			go func() {
```

Comment:
Give we do not retrieve asynchronously we could simplify this code and get rid of channels and goroutines.

Reply:
Otherwise looks good.